runner/app/main.py

```diff
@@ -21,8 +21,6 @@ async def lifespan(app: FastAPI):
     app.pipeline = load_pipeline(pipeline, model_id)
     app.include_router(load_route(pipeline))
 
-    use_route_names_as_operation_ids(app)
-
     logger.info(f"Started up with pipeline {app.pipeline}")
```

Comment:
I still need to update the https://github.com/livepeer/ai-worker/blob/main/runner/gen_openapi.py file but this has low priority. We currently don't use the model deployment.